development/docker-compose/docker-compose.yaml

```diff
@@ -153,10 +153,10 @@ services:
       JAVA_TOOL_OPTIONS: "-Dquarkus.log.category.'\"com.backbase\"'.level=DEBUG \
                           -Dquarkus.log.category.'\"org.keycloak.protocol.oidc\"'.level=TRACE"
     volumes:
-      - ./scripts/HealthCheck.java:/tmp/HealthCheck.java
+      - ./exe/HealthCheck.jar:/tmp/HealthCheck.jar
```

Comment:
this does not work for identity, the .java file works as it is still on java 17

Reply:
even though the base [image](https://github.com/Backbase/local-backend-setup/pull/16/files#diff-b77f5795e92f51483c4e21d687301e7eb9c9976ea76d32367bf099b7bb0bc11fR39) is with java21?

Reply:
yes , i changed it to java 21

Reply:
the problem was in my local